tests/test_octodns_record.py

```diff
@@ -755,17 +755,104 @@ def test_A_and_values_mixin(self):
             'ttl': 600,
             'values': [
                 '1.2.3.4',
+            ]
+        })
+        Record.new(self.zone, '', {
+            'type': 'A',
+            'ttl': 600,
+            'values': [
+                '1.2.3.4',
                 '1.2.3.5',
             ]
         })
 
-        # missing value(s)
+        # no values, no value or values in data
+        a = ARecord(self.zone, '', {
```

Comment:
Not sure if this is an appropriate place for these test.

It's testing the _data method of ValuesMixin, but not actually a Validation. This is under TestValidation class

Reply:
Don't have strong opinions on whether they should live here or in `TestRecord`. If you don't feel like they fit with validations it'd probably make sense to move them up there. Probably worth sticking them in a dedicated function, `def test_data` or similar.